cmd/appDeploy.go

```diff
@@ -45,6 +45,14 @@ var deployExample = `
   # Deploy an app using app-name, container image and pass environment variables and set port where application listens on.
   appctl deploy -n <appname> -i <image> -e key1=value1 -e key2=value2 -p <port>
   Ex: appctl deploy -n hello -i gcr.io/knative-samples/helloworld-go -e TARGET="appctler" -p 7893
+
+  # Deploy an app using app-name, container image and pass environment variables through a file and set port where application listens on.
+  appctl deploy -n <appname> -i <image> -f <env-file-path> -p <port>
+  Ex: appctl deploy -n hello -i gcr.io/knative-samples/helloworld-go -f /Users/user/variables.env -p 7893
+
+  # Deploy an app using app-name, container image and pass environment variables through a file or pass through command line and set port where application listens on.
```

Comment:
I think its ... pass environment variables through a file `and` pass through command line instead of `or`